user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md

```diff
@@ -92,6 +103,8 @@ To configure TLS encryption for client-server communication:
 1. Add the full *https://* URL (including the port) in the \<DBServer> element in the *DB.xml* file. For example:
    `<DBServer>https://elastic.dataminer:9200</DBServer>`
 
+1. Add the certificates on each Dataminer agent under `Trusted root certification authorities`.
```

Comment:
Import the certificate in the 'Trusted Root Certification Authorities' store of the Windows Certificate store.